portfolio/src/main/java/com/google/sps/data/Game.java

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.Arrays;
+import java.util.List;
+import java.lang.*; 
+
+
+//Class representing the two truths and one lie game.
+public class Game {
+    private String[] truths = new String[] {"I play the piano.", 
+                    "I don\'t like pineapples on pizza.",
+                    "I have never been skydiving before.", 
+                    "I am hungry.", 
+                    "I visited my friend in Manchester last Christmas."};
+
+    private String[] lies = new String[] {"My favourite subject in high school was Calculus.",  
```

Comment:
I don't see this get modified anywhere. Use constants (static finals) and rename the field to CONSTANT_CASE

Reply:
Okay!